src/epcpm/cantoxlsx.py

```diff
@@ -403,6 +463,10 @@ def format_for_manual(
         for group_parameter_filter in FILTER_GROUPS:
             if parameter_path.startswith(PARAMETERS_PREFIX + group_parameter_filter):
                 filter_out = True
+        # Filter out specific nodes in FILTER_NODES list.
+        for node_parameter_filter in FILTER_NODES:
```

Comment:
I think we cannot just use "After" and "Before" to filter out parameters from the list. There are still parameters in parameter tree using "After" and "Before" that are relevant:
![image](https://github.com/user-attachments/assets/3affdb40-a090-416c-9f8d-dcfd056a434e)
With Dorian's scripts these were included in the output:
![image](https://github.com/user-attachments/assets/c035a335-0f0d-4276-b9d4-3e4b19a22df8)
I tested the script in this PR and these are missing.

I think the decisive factor is the path to parameters:
AC -> C. Grid Support -> Hz-Watt -> Tables -> After (exists in CAN but not relevant since no getters/setters
AC -> C. Grid Support -> Hz-Watt -> Tables -> Tree -> 1 -> After (relevant parameters)

Reply:
Fixed in this commit: 6c1688037b416b9a52a8535b6c3f58c703eec6be